CHANGELOG.md

```diff
@@ -57,8 +64,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Initial release to PyPI
 
-[Unreleased]: https://github.com/nathanegillett/chexus/compare/v2.1.0...HEAD
+[Unreleased]: https://github.com/nathanegillett/chexus/compare/v2.2.0...HEAD
```

Comment:
This isn't accurate.

```suggestion
[Unreleased]: https://github.com/nathanegillett/chexus/compare/v2.1.0...HEAD
```

Reply:
Ahh, I misunderstood the purpose of this. Thanks!